### PR TITLE
Feature/rework integration example

### PR DESCRIPTION
**Pull request by Milena-Czierlinski**

### CODE REVIEW

File: _includes/scenarios/integrate/integration-example.md [hunk 1]
```diff
@@ -1,151 +1,114 @@
...
+You need to replace them with values before you send the requests.
 
 ## Prerequisites
 
 - If you want to use your own Connector for executing the examples:
   - [Install the Connector](https://enmeshed.eu/integrate/connector-installation)
-  - Make sure the [Sync Module is disabled](https://enmeshed.eu/integrate/connector-configuration#sync) (because in this tutorial we will synchronize manually via the HTTP endpoint)
-  - Make sure the [docs are enabled](https://enmeshed.eu/integrate/connector-configuration#corehttpapi) for the documentation route to work
+  - Make sure the [Sync Module is disabled](https://enmeshed.eu/operate/configuration#sync) (because in this tutorial we will synchronize manually via the HTTP endpoint)
+  - Make sure the [docs are enabled](https://enmeshed.eu/operate/configuration#corehttpapi) for the documentation route to work
```
Comment (jkoenig134) [inline _includes/scenarios/integrate/integration-example.md:17]: please generate these links

File: _includes/scenarios/integrate/integration-example.md [hunk 2]
```diff
@@ -1,151 +1,114 @@
...
+| `attribute`               | `<IdentityAttribute created in the previous step>`     |
+| `sourceAttributeId`       | `"<id of the attribute created in the previous step>"` |
+| `mustBeAccepted`          | `true`                                                 |
+
+| ReadAttributeRequestItems |                              |
+| ------------------------- | ---------------------------- | ---------------------------- | ---------------------------- |
+| `@type`                   | `"ReadAttributeRequestItem"` | `"ReadAttributeRequestItem"` | `"ReadAttributeRequestItem"` |
+| `query.@type`             | `"IdentityAttributeQuery"`   | `"IdentityAttributeQuery"`   | `"IdentityAttributeQuery"`   |
+| `query.valueType`         | `"GivenName"`                | `"Surname"`                  | `"EMailAddress"`             |
+| `mustBeAccepted`          | `true`                       | `true`                       | `false`                      |
```
Comment (jkoenig134) [inline _includes/scenarios/integrate/integration-example.md:79]: can you explain why these tables are better than the json of the old version? 

cc @stnmtz 

1. the reader of this guide HAS TO HAVE json knowledge anyway
2. with these tables the reader has to transform the tables to json, this is a massive overhead

Comment (jkoenig134) [reply _includes/scenarios/integrate/integration-example.md:79]: same for every table

Comment (vinzlange) [reply _includes/scenarios/integrate/integration-example.md:79]: this is the scenario that should be possible even with very little knowledge. we may have to find an alternative here.

Comment (jkoenig134) [reply _includes/scenarios/integrate/integration-example.md:79]: this "scenario" is the main entrance to integrating into the enmeshed world. knowing how to read json should be the "very little knowledge" we should assume the user to have.

Comment (jkoenig134) [reply _includes/scenarios/integrate/integration-example.md:79]: IMO understanding how to transform these tables to JSON is **so much harder** than just knowing how to read JSON.

File: _includes/scenarios/integrate/integration-example.md [hunk 3]
```diff
@@ -1,151 +1,114 @@
...
-```
-
-You can query the Connector's Address under the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the Address is `id134nJmN7E4Carb6KyRJyePVnXxVHEYQgWD`.
-{: .notice--info}
+Firstly, we want to create a display name for our Connector.
+While communicating with the other Identity in the subsequent steps, we will choose to share this display name.
+Technically, for this we need to create an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) with an IdentityAttribute Value of type [DisplayName]({% link _docs_integrate/attribute-values.md %}#displayname) for our Connector.
+To do so, proceed as described in the Create own IdentityAttribute guide with the following content.
 
-{% include rapidoc api_route_regex="^post /api/v2/Attributes$" %}
```
Comment (jkoenig134) [inline _includes/scenarios/integrate/integration-example.md:44]: why are the snippets removed? this guide should be self-contained, now it's a jump-jump-jump-jump-jump-jump

Comment (vinzlange) [reply _includes/scenarios/integrate/integration-example.md:44]: we only wanted to use rapidocs in usecases, not in scenarios

Comment (jkoenig134) [reply _includes/scenarios/integrate/integration-example.md:44]: (IMO)

Comment (jkoenig134) [reply _includes/scenarios/integrate/integration-example.md:44]: maybe we have to move this out of the scenarios to make this clear.

Comment (vinzlange) [reply _includes/scenarios/integrate/integration-example.md:44]: taking out of scenarios sounds promising, we need to discuss this with @stnmtz

**Comment by Milena-Czierlinski** [comment]
Since we discussed not to change the JSON payloads to tables and leave the rapidocs, which is the main part of this PR, should I close this PR, such that we can start from scratch wanting to improve the Integration Example and such that it doesn't block other PRs?

**Comment by jkoenig134** [comment]
I don't think this blocks anything. Just assume that it isn't here :D

**Comment by vinzlange** [comment]
> I don't think this blocks anything. Just assume that it isn't here :D

Somehow it blocks the new excel update script, because of the _include structure. Or becomes faulty if the new script is used

**Comment by jkoenig134** [comment]
yeah, it will get some merge conflicts, but this is ok. should be easy to resolve.

**Review by jkoenig134 — APPROVED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**